Recruit-Training/Advanced-Recruit-Training/Telemetry-Teaser/src/app/app.component.html

```diff
@@ -16,10 +19,13 @@ <h1> Elysia Range Simulator </h1>
       <br>
       <label>Battery Percentage (%):</label>
       <br>
-      <input class="form-control"  [ngClass]="{
-	  'invalid-field': inputBattery.invalid && (inputBattery.dirty || inputBattery.touched),
-	  'valid-field': inputBattery.valid && (inputBattery.dirty || inputBattery.touched)
-	  }" name="battery" formControlName="inputBattery" type="number" placeholder="Battery" (keyup)="onKeyBattery($event)" >
+      <input [ngClass]="{
+		  'invalid-field': inputBattery.invalid && (inputBattery.dirty || inputBattery.touched),
+		  'valid-field': inputBattery.valid && (inputBattery.dirty || inputBattery.touched),
+		  'form-control': inputBattery.prestine
+		}"
+		name="battery" formControlName="inputBattery" type="number" placeholder="Battery" 
+	(keyup)="onKeyBattery($event)" >
```

Comment:
Sorry I meant lines 27-28 were supposed to be on the same line. The comment just made it look like two lines.